src/ctia/bulk/core.clj

```diff
@@ -99,16 +93,13 @@
 (s/defn read-entities
   "Retrieve many entities of the same type provided their ids and common type"
   [ids entity-type auth-identity
-   services :- ReadEntitiesServices]
-  (let [read-entity (read-fn entity-type auth-identity {} services)]
-    ;; TODO search by ids
-    (pmap (fn [id]
-           (try
-             (some-> (read-entity id)
-                     (with-long-id services))
-             (catch Exception e
-               (log/error (pr-str e)))))
-         ids)))
+   {{:keys [get-store]} :StoreService
+    :as services} :- ReadEntitiesServices]
+  (when-let [store (get-store entity-type)]
```

Comment:
I think `get-store` throws when it can't find a store for a particular entity type. And I added when-let to cover one specific test, so I will remake the test :)